docs/api.md

```diff
@@ -254,14 +254,18 @@ Enables printing of level labels instead of level values in the printed logs.
 Warning: this option may not be supported by downstream transports.
 
 <a id="changeLevelName"></a>
-#### `changeLevelName` (String)
+#### `changeLevelName` (String) - DEPRECATED
+Use `levelKey` instead. This will be removed in v6.
```

Comment:
I added this as a placeholder. Comments/edits welcome.

Reply:
Deprecation would last _through_ v6. So it would be removed in v7.